code/modules/clothing/glasses/glasses.dm

```diff
@@ -335,6 +362,39 @@ BLIND     // can't see anything
 	sprite_state = "idris_goggles"
 	brand_name = "Idris"
 
+//medical hud
+/obj/item/clothing/glasses/safety/goggles/medical
+	name = "medical goggles"
```

Comment:
```suggestion
	name = "medical HUD goggles"
```

Reply:
I'll add the fact that it has an HUD to its description as a few other items do. It won't be put into the name, though.